packages/dapp/src/components/PaymentDetailsForm.jsx

```diff
@@ -144,7 +145,7 @@ export const PaymentDetailsForm = ({ display, tokenData, allTokens }) => {
             );
             setMilestonesInvalid(isNaN(Number(v)) || Number(v) === 0);
           }}
-          tooltip="Number of milestones in which the total payment will be processed"
+          tooltip="How many milestone payments will there be for this invoice? (You’ll be able to customize the payment amount for each milestone later)."
```

Comment:
Better to state in the parenthesis: (You'll be able to customize the payment amount for each milestone in the next step).